Description: int narrowing warning fix
 On some architectures, there's a compilation warning about int
 being narrowed to char, when compiling with g++ version 5. Since
 warnings are treated as errors (-Werror compilation flag), on
 those architectures Kadu does not compile.
Author: Patryk Cisek <patryk@debian.org>
---
diff --git a/plugins/jabber_protocol/3rdparty/libiris/src/xmpp/base64/base64.cpp b/plugins/jabber_protocol/3rdparty/libiris/src/xmpp/base64/base64.cpp
index 0269c1e..ba9c600 100644
--- a/plugins/jabber_protocol/3rdparty/libiris/src/xmpp/base64/base64.cpp
+++ b/plugins/jabber_protocol/3rdparty/libiris/src/xmpp/base64/base64.cpp
@@ -45,7 +45,7 @@ QByteArray Base64::decode(const QString& input)
 	// 64 specifies eof
 	// everything else specifies data
 
-	char tbl[] = {
+	int tbl[] = {
 		-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,
 		-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,
 		-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,62,-1,-1,-1,63,
@@ -80,10 +80,10 @@ QByteArray Base64::decode(const QString& input)
 	c = d = 0;
 
 	for( i = 0; i < len; i += 4 ) {
-		a = tbl[(int)s[i]];
-		b = tbl[(int)s[i + 1]];
-		c = tbl[(int)s[i + 2]];
-		d = tbl[(int)s[i + 3]];
+		a = tbl[s[i]];
+		b = tbl[s[i + 1]];
+		c = tbl[s[i + 2]];
+		d = tbl[s[i + 3]];
 		if((a == 64 || b == 64) || (a < 0 || b < 0 || c < 0 || d < 0)) {
 			p.resize(0);
 			return p;
